stable/yugabyte/values.yaml

```diff
@@ -8,7 +8,7 @@ nameOverride: ""
 
 Image:
   repository: "yugabytedb/yugabyte"
-  tag: 2.19.3.0-b140
```

Comment:
please don't modify the values.yaml for this, this is modified by generated code.